src/components/Header/Header.js

```diff
@@ -0,0 +1,32 @@
+import { MDCTopAppBar } from "@material/top-app-bar";
+import headerTemplate from "./Header.html";
+
+export class HeaderComponent {
+  constructor(mountPoint, props = {}) {
+    this.mountPoint = mountPoint;
+    this.props = props;
+  }
+
+  initMaterial() {
+    this.topAppBar = MDCTopAppBar.attachTo(this.mountPoint);
+  }
+
+  setScroll() {
```

Comment:
actually it seems to be the part of material initialization. Just move contents of this method to initMaterial